src/modules/dashboard/components/NetworkContractUpgradeDialog/NetworkContractUpgradeDialogForm.tsx

```diff
@@ -0,0 +1,147 @@
+import { ColonyRole } from '@colony/colony-js';
+
+import Button from '~core/Button';
+import DialogSection from '~core/Dialog/DialogSection';
+import { Annotations } from '~core/Fields';
+import Heading from '~core/Heading';
+import PermissionsLabel from '~core/PermissionsLabel';
+import PermissionRequiredInfo from '~core/PermissionRequiredInfo';
+
+import { Colony, useLoggedInUser, useNetworkContracts } from '~data/index';
+import { useTransformer } from '~utils/hooks';
+
+import { getAllUserRoles } from '../../../transformers';
+import { hasRoot } from '../../../users/checks';
+
+import { FormValues } from './NetworkContractUpgradeDialog';
+import styles from './NetworkContractUpgradeDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.title',
+    defaultMessage: 'Upgrade Colony',
+  },
+  currentVersion: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.currentVersion',
+    defaultMessage: 'Current version',
+  },
+  newVersion: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.newVersion',
+    defaultMessage: 'New version',
+  },
+  annotation: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.annotation',
+    defaultMessage: 'Explain why you are upgrading this colony (optional)',
+  },
+  noPermissionText: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.noPermissionText',
+    defaultMessage: `You do not have the {requiredRole} permission required
+      to take this action.`,
+  },
+});
+
+interface Props {
+  back: () => void;
+  colony: Colony;
+}
+
+const NetworkContractUpgradeDialogForm = ({
+  back,
+  colony,
+  colony: { version },
+  handleSubmit,
+  isSubmitting,
+}: Props & FormikProps<FormValues>) => {
+  const { walletAddress, username, ethereal } = useLoggedInUser();
+
+  const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
+
+  const hasRegisteredProfile = !!username && !ethereal;
+  const canUpgradeVersion = hasRegisteredProfile && hasRoot(allUserRoles);
+
+  const { version: newVersion } = useNetworkContracts();
+
+  return (
+    <>
+      <DialogSection appearance={{ theme: 'heading' }}>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none' }}
+          text={MSG.title}
+          className={styles.title}
+        />
+      </DialogSection>
+      {!canUpgradeVersion && (
+        <DialogSection>
+          <PermissionRequiredInfo requiredRoles={[ColonyRole.Root]} />
+        </DialogSection>
+      )}
+      <DialogSection>
+        <div className={styles.contractVersionLine}>
```

Comment:
As I've said previous,ly please reduce the left/right padding, so that the version lines span the whole width of the modal.

Yours:
![Screenshot from 2021-01-12 18-17-05](https://user-images.githubusercontent.com/1193222/104341776-d82c2780-5502-11eb-8e0d-5459e378d2f8.png)

How I want this to look:
![Screenshot from 2021-01-12 18-16-43](https://user-images.githubusercontent.com/1193222/104341817-e24e2600-5502-11eb-937d-566561a13224.png)